README.MD

```diff
@@ -15,21 +15,27 @@ This is the source code behind [blog.returnnull.de](https://blog.returnnull.de).
 [![PHPStorm](code/public/img/stack/phpstorm.png)](https://www.jetbrains.com/de-de/phpstorm/)
 [![Git](code/public/img/stack/git.png)](https://git-scm.com/)
 
+## How to Start Development
+- Clone repository or Fork
+- Build images
+  - Build with: `make build`
+  - Building with Docker: `docker compose build`
```

Comment:
Both commands use docker, so the wording should be:

- Build images
  - Build with: `make build` or with `docker compose build`